src/lib/stark/ledger/createSignedOrder.js

```diff
@@ -1,13 +1,20 @@
 const Eth = require('@ledgerhq/hw-app-eth').default
-const byContractAddress = require('@ledgerhq/hw-app-eth/erc20')
-  .byContractAddress
+const { byContractAddress } = require('@ledgerhq/hw-app-eth/erc20')
 const DVFError = require('../../dvf/DVFError')
 const BN = require('bignumber.js')
 const _ = require('lodash')
 const selectTransport = require('../../ledger/selectTransport')
 const getTokenAddressFromTokenInfoOrThrow = require('../../dvf/token/getTokenAddressFromTokenInfoOrThrow')
 
-module.exports = async (dvf, path, starkOrder) => {
+const getPublicKey = async (eth, starkPath) => {
+  const tempKey = (await eth.starkGetPublicKey(starkPath)).toString('hex')
+  return {
+    x: tempKey.substr(2, 64),
+    y: tempKey.substr(66)
+  }
+}
+
+module.exports = async (dvf, path, starkOrder, { dontGetPublicKey = false } = {}) => {
```

Comment:
Do we want to positivize `dontGetPublicKey` to `getPublicKey` ?

Reply:
fixed in 09978ab60df1bb0707e7f20ffbbdcff3d694d491